### ISSUE TITLE
Renaming global directory

### ISSUE THREAD

**Issue by reicda** [OpenSlides/openslides-backend#2604]
At the moment we use global as the name for the directory containing the meta repository and data intended for further use in other modules.

'global' is one of the [reserved keywords in python](https://docs.python.org/3/reference/lexical_analysis.html#keywords) and thus must not be used in this case as it will lead to a syntax error. There are work-arounds but they will lead to errors by the linting tools we use (e.g. mypy) that need to be suppressed.

example: 
`>>> import global.example_module`

work-around:
`>>> sys.path.append("global")`
`>>> import example_module`

work-around (suppressed linting):
`>>> sys.path.append("global")`
`>>> import example_module # type: ignore # noqa`

To tackle the problem at the root I suggest we rename the directory to something similar (e.g. universal, general, unbounded, planetary, comprehensive) and adjust all references.

**Comment by hjanott** [comment]
> Is it possible for the backend to moce the folders (plus files in these folders) one level up and delete the "global" folder alltogether?

Yes it's possible. But it will also require some changes to the main repository as well as to the client and auth-service.

**Comment by peb-adr** [comment]
We will also need to check where the manage service might depend on the `global/data` path.

And possibly more services..

**Comment by peb-adr** [comment]
> Is it possible for the backend to moce the folders (plus files in these folders) one level up and delete the "global" folder alltogether?

This proposal I like.
It would also make the `meta` path more unified. Comparing to autoupdate and search service, which also include `meta` at the root of their file tree, the backend is one exception using `global/meta`.
The client ofc still falls out of this, but there moving the directory has other implications and is not as easy.

**Comment by peb-adr** [comment]
> We will also need to check where the manage service might depend on the `global/data` path.
> 
> And possibly more services..

Checked the manage service.
It does **not** directly depend on the paths. It just calls `organization.initial_import`.
